configs/data_gatherer.conf

```diff
@@ -18,8 +18,10 @@
 --carla_camera_image_height=600
 #--nocarla_random_player_start
 #--carla_start_player_num=1
+--perfect_tracking
+--perfect_tracking_num_steps=10
```

Comment:
Can you please also update data_gatherer_autopilot.conf?

Reply:
Done.